build/configs/testing_config.yaml

```diff
@@ -6,7 +6,7 @@ ethereum:
       password: ''
       address: 89b0a86583c4444acfd71b463e0d3c55ae1412a5
 
-identityId: "0x010101010101"
+identityId: "0x89b0a86583c4444acfd71b463e0d3c55ae1412a5"
```

Comment:
Is it a dummy address or do we need a real identity contract for the testing_config?

Reply:
It happens to be a real address, as I took it from the eth_account, but I believe that it doesnt get used.